client/src/ui/organisms/footer.tsx

```diff
@@ -1,22 +1,21 @@
 import * as React from "react";
 
-import { useLocale } from "../../hooks";
-import Logo from "../atoms/logo";
-
 import { ReactComponent as GithubIcon } from "@mdn/dinocons/brands/github-mark-small.svg";
 import { ReactComponent as TwitterIcon } from "@mdn/dinocons/social/twitter.svg";
 import { ReactComponent as InstagramIcon } from "@mdn/dinocons/social/instagram.svg";
 
+import { useLocale } from "../../hooks";
+import { Logo } from "../atoms/logo";
 import "./footer.scss";
 
-export default function Footer() {
+export function Footer() {
   const locale = useLocale();
   return (
     <footer id="nav-footer" className="page-footer">
       <div className="content-container">
-        <a href={`/${locale}/`} className="page-footer-logo">
+        <div className="page-footer-logo">
```

Comment:
Perhaps this isn't ideal. I just noticed your SCSS file expected there to be a `.page-footer-logo .logo svg` hierarchy. Perhaps that could be simplified with `.page-footer .logo` and then we don't even need this extra div.

Reply:
Well, I guess scratch that. The `.page-footer-logo` is not just used for setting the `fill: #fff;` but for all sorts of other grid stuff.